examples/flutter_gallery/lib/demo/playground/cupertino/segment_control.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import '../configuration/material_helpers.dart';
+import '../playground_demo.dart';
+
+const String _demoWidgetName = 'CupertinoSegmentControl';
+
+/// ignore: must_be_immutable
```

Comment:
This lint can't be ignored. The fact that widgets are stateless is an invariant that Flutter relies on.

Reply:
What is a possible solution here? Thanks.

Reply:
Sadly, there are loads of conflicts. See https://github.com/flutter/flutter/pull/24764#issuecomment-442526597